src/zadaci/zadatak2/Bookshelf.java

```diff
@@ -0,0 +1,38 @@
+package zadaci.zadatak2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bookshelf {
+    private List<zadaci.zadatak2.Book> listOfBooks;
```

Comment:
you have some copy pasted classes, so you need to import from a specific package, delete them it is easier to read